src/common/common-util/src/main/java/ee/ria/xroad/common/conf/globalconf/VersionedConfigurationDirectory.java

```diff
@@ -276,6 +320,16 @@ public static ConfigurationPartMetadata getMetadata(Path fileName) throws IOExce
         }
     }
 
+    public static boolean isVersion3(Path filePath) {
```

Comment:
This method name is tied to `CURRENT_GLOBAL_CONFIGURATION_VERSION`  value and should be renamed when value changes. Maybe rename it `isCurrentVersion`?